pkg/gui/types/views.go

```diff
@@ -34,6 +34,10 @@ type Views struct {
 	AppStatus         *gocui.View
 	Search            *gocui.View
 	SearchPrefix      *gocui.View
+	StatusSpacer1     *gocui.View
```

Comment:
Do we actually need to create views for this? I feel like it should be possible to have the window arrangement code add spacers but where there's no actual underlying view there

Reply:
If it were just for the window arrangement, then I'd agree, but we need the views to actually draw the space. I didn't find a way to make gocui draw things except by having a view that does it.

Reply:
Why do we need the view to draw the space? My assumption is that the view would be confined to its own frame so you'd get the space naturally if there's a one character gap between views

Reply:
huh: whaddayathink about having gocui track which cells aren't in a view and clearing them on each screen draw?

Reply:
Hm, I had hoped not to have to do such fundamental changes to gocui as part of this. I'm not too bothered by having to instantiate these spacer views, to be honest.

Also, it wouldn't be totally obvious how to implement what I'm doing in the followup PR, where I pass all the views of the bottom line (including the spacer views) to gocui to redraw. I'd probably have to additionally pass a rectangle to clear, or something like that. I'm not sure that's really so much better.